ghost/admin/app/components/koenig-lexical-editor.js

```diff
@@ -669,34 +669,53 @@ export default class KoenigLexicalEditor extends Component {
         const multiplayerDocId = cardConfig.post.id;
         const multiplayerUsername = this.session.user.name;
 
+        const [hasInitialised, setHasInitialised] = React.useState(false);
+
+        const KGEditorComponent = ({isInitInstance}) => {
+            const handleInitInstance = (data) => {
+                if (this.args.initLexical && isInitInstance && !hasInitialised) {
+                    this.args.initLexical(data);
+                    setHasInitialised(true);
```

Comment:
Why do we need `hasInitialised` here? One of the issues Steve ran into is that the changes made when opening the editor come in batches so we can't guarantee that only the first `onChange` call is the one we want - if it was then we could do the same thing without having a secondary instance at all